### PR TITLE
fix: avatar upload size error

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR I fix an issue where the user avatar picture was artificially growing in size before being sent to the backend. This resulted in a silent error and failure to update the avatar picture. With this PR I update the code that handles cropping and creating a new image. It uses lossy compression to reduce the file size and ensure we do not hit the limit of 10Mb that we fixed for file uploads.

Maybe this component should be moved to UI alongside a custom hook to compute the image crop. @ReidyT were you working on that recently ?

### CODE REVIEW

**Comment by ReidyT** [comment]
@spaenleh I started the move to the UI, but as I thought it wasn't a priority, I haven't finished and scheduled for later. It is urgent for now ?

**Comment by spaenleh** [comment]
@ReidyT No not urgent. Just wanted to know what the status of that was, so I could maybe add this fix there and we would not have to duplicate the code too much. Lets discuss this.

File: src/components/main/CropModal.tsx [hunk 1]
```diff
@@ -74,51 +78,71 @@ const CropModal = ({ onConfirm, onClose, src }: CropProps): JSX.Element => {
...
-    // at the expense of slightly slower render times and needing to
-    // size the image back down if you want to download/upload and be
-    // true to the images natural size.
-    const pixelRatio = window.devicePixelRatio;
-    // const pixelRatio = 1
+    /**
+     * Compute the relative width and height
+     * They express the size of the crop in terms of a relative portion of the original image.
+     * For example if only the first quarter of the image was used in the crop,
+     * the relativeCrop would be 0.5, 0.5, meaning half of the height, abd half of the width
```
Comment (ReidyT) [inline src/components/main/CropModal.tsx:85]:
```suggestion
     * the relativeCrop would be 0.5, 0.5, meaning half of the height, and half of the width
```

**Review by ReidyT — APPROVED** [review]
LGTM, just a small issue in a comment 🚀

**Pull request merged**